### PR TITLE
FIX Caret was not visible in amount textinput

### PR DESCRIPTION
**Pull request by marcosrdz**
## Summary by CodeRabbit

- **New Features**
	- Improved cursor positioning in the Amount Input field for a better user experience during text entry.

### CODE REVIEW

File: components/AmountInput.js [hunk 1]
```diff
@@ -211,6 +211,13 @@ class AmountInput extends Component {
     });
   };
 
+  handleSelectionChange = event => {
+    const { selection } = event.nativeEvent;
+    if (selection.start !== selection.end || selection.start !== this.props.amount?.length) {
+      this.textInput?.setNativeProps({ selection: { start: this.props.amount?.length, end: this.props.amount?.length } });
+    }
+  };
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply components/AmountInput.js:219]: Nah. Thats exactly the caret behavior needed

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**